### ISSUE TITLE
Color contrast issue

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#689]
### What happened?

The 'stage' title doesn't have enough contrast. Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds.

To solve this problem, you need to fix the following:
Element has insufficient color contrast of 2.96 (foreground color: #ffffff, background color: #ff640a, font size: 16.0pt (21.3333px), font weight: bold). Expected contrast ratio of 3:1

### Anything else?

![Image](https://github.com/user-attachments/assets/baa118e1-5299-409b-9fd7-5ba70f51c1c3)

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows
